src/Okta.Idx.Sdk/IdxClient.cs

```diff
@@ -266,6 +266,35 @@ internal async Task<IIdxResponse> IntrospectAsync(IIdxContext idxContext, Cancel
                 request, cancellationToken).ConfigureAwait(false);
         }
 
+        /// <inheritdoc/>
+        public async Task<IdentityProvidersResponse> GetIdentityProvidersAsync(string state = null, CancellationToken cancellationToken = default)
+        {
+            IIdxContext idxContext = await this.InteractAsync(state);
+            return await GetIdentityProvidersAsync(idxContext, cancellationToken);
+        }
+
+        /// <inheritdoc/>
+        public async Task<IdentityProvidersResponse> GetIdentityProvidersAsync(IIdxContext idxContext, CancellationToken cancellationToken = default)
+        {
+            IIdxResponse introspectResponse = await this.IntrospectAsync(idxContext);
+
+            return new IdentityProvidersResponse
+            {
+                Context = idxContext,
+                IdpOptions = introspectResponse.Remediation?.RemediationOptions?
+                    .Where(remediationOption => remediationOption.Name.Equals("redirect-idp"))

```

Comment:
We have an enum for remediation names, can we please add `redirect-idp` there so we can reuse it?